tests/pingpong-shared/pingpong.edl

```diff
@@ -1,18 +1,20 @@
+
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
 enclave {
     trusted {
         public void Ping(
             [in, string] const char* in,
-            [in, out, string] char* out);
-
+            [in, out, string] char* out,
+			int out_length);
     };
 
     untrusted {
         void Pong(
             [in, string] const char* in,
-            [in, out, string] char* out);
+            [in, out, string] char* out,
+			int out_length);
```

Comment:
this should be marked wit the [in] annotation

Reply:
There would be "unexpected pointer attributes for `int'" error after I added it.

Reply:
oh, actually that makes sense. An annotation wouldn't be needed for integral types. Go ahead and ignore :smile: